edgedb-errors/src/error.rs

```diff
@@ -54,6 +57,9 @@ impl Error {
     pub fn initial_message(&self) -> Option<&str> {
         self.0.messages.first().map(|m| &m[..])
     }
+    pub fn contexts(&self) -> Iter<Cow<'static, str>> {
```

Comment:
Since this is a public API visible to users let's take a more conservative approach:
```suggestion
    pub fn contexts(&self) -> impl Iterator<Item=&str> {
```
(this will require a map, and maybe lifetime on a function, but shouldn't be hard)